src/backend/api/Fusion.Resources.Domain/Events/ResourceAllocationRequestEventTypes.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System.Collections.Generic;
+
+namespace Fusion.Events
+{
+    public static class ResourceAllocationRequestEventTypes
```

Comment:
These models should not be defined in the domain model.. maybe they could be added to the Integration.Models?

Reply:
Agree. Moved to Integration.Models